tables/isolate_mutations.csv

```diff
@@ -4490,3 +4490,13 @@ B.1.351 Spike:-18F+241del-244del-246I,S,484,K
 B.1.351 Spike:-18F+241del-244del-246I,S,501,Y
 B.1.351 Spike:-18F+241del-244del-246I,S,614,G
 B.1.351 Spike:-18F+241del-244del-246I,S,701,V
+B.1.351 Spike:-246I-614G,S,18,F

```

Comment:
For pseudovirus, we prefer to use whole Spike mutation as the iso_name, like S:18F+501Y+614G. You can change B.1.351 Spike:-246I-614G to the "S form".